src/nublado/controller/config.py

```diff
@@ -730,6 +739,15 @@ class LabInitContainer(BaseModel):
 
     image: Annotated[ContainerImage, Field(title="Image to run")]
 
+    command: Annotated[
+        list[str] | None,
+        Field(
+            title="Command and arguments for container",
+            description=("Set container command and arguments."),
+            examples=[["nublado", "purger", "warn"]],
```

Comment:
Does this example make sense? I'm not sure what running the purger in an init container would do. Maybe one of the commands we already run in an init container would be a better example?